plotsquared/commands.md

```diff
@@ -0,0 +1,1438 @@
+# Commands
+
+## Introduction
+
+All commands start with `/plot` if not configured else wise.
+
+As alternative or short-command you can use the following aliases:
+`plots, p, plotsquared, p2, ps, 2, plotme`
+
+## Command Overview
+
+PlotSquared have an amount of commands for different uses. At here you find the complete command list with her syntax and permissions.
+
+command-categories:
+
+* [Basics](/plotsquared/commands#basics)
+* [Info](/plotsquared/commands#info)
+* [Teleport](/plotsquared/commands#teleport)
+* [Chat](/plotsquared/commands#chat)
+* [Claiming](/plotsquared/commands#claiming)
+* [Settings](/plotsquared/commands#settings)
+* [Schematic](/plotsquared/commands#schematic)
+* [Rating](/plotsquared/commands#rating)
+* [Administration](/plotsquared/commands#administration)
+* [Debug](/plotsquared/commands#debug)
+* [Other_permissions](/plotsquared/commands#other_permissions)
```

Comment:
Clicking on the hyperlink should link within the document on gitbook. Currently, it links to GitHub.

Reply:
@NotMyFault Are you sure that's not the correct link at GitBook after building the site?

Reply:
I clicked on one of the items and was on GitHub. The format shown to utilize links doesn't match what you put in place.

Reply:
Hopefully it works now.